src/intl/messages.js

```diff
@@ -574,7 +575,7 @@ export const messages: { [messageId: string]: MessageType } = {
   },
   resetSettings: 'Reset settings',
   resetSettingsQuestion: 'Are you sure you want to reset your account settings?',
-  resetSettingsWarning: 'Resetting your account settings will clear all of the individual settings that were applied to it. Your account settings will be returned to the default settings. Please confirm that you want to reset your account settings before making these changes.',
+  resetSettingsWarning: 'The reset will apply the system defaults to ALL of your settings across all settings tabs. Please confirm you want to fully reset your account settings.',
```

Comment:
I wouldn't touch this original message as rephrased by Laura back then since it will require re-translation for all languages which we currently try to avoid if not necessary.
If you think that leaving this long text and just adding a new clarification sentence `additionalNote` regarding applying to all settings tabs is too long and not essential then let's leave it as is, otherwise let's just add this `additionalNote`. I don't have a strong opinion on that

Reply:
Okay, I am moving the original message back.